kie-wb/pom.xml

```diff
@@ -33,6 +33,7 @@
   <modules>
     <module>kie-wb-webapp</module>
     <module>kie-wb-distribution-wars</module>
+    <module>kie-wb-webapp-test</module>
```

Comment:
Maybe we could move this outside of the `kie-wb` submodule? I think it would be great to have the possibility to run the tests both on `kie-wb` and `kie-drools-wb` since lot of the functionality is in both.